### PR TITLE
Story 11: Game displays a welcome statement

### PR DESCRIPTION
**Pull request by ericdrosado**

### CODE REVIEW

File: spec/game_spec.rb [hunk 1]
```diff
@@ -1,7 +1,23 @@
-require_relative '../game'
+#require_relative '../game'
+require_relative 'mocks/mock_game'
+require_relative 'mocks/mock_view'
+
 
 describe 'Game' do
+  mock_view = MockView.new
+  prompter = Prompter.new
+  mock_game = MockGame.new(prompter, mock_view)
```
Comment (NicoleCarpenter) [inline spec/game_spec.rb]: This is wrong. If you are testing the game, you have to test the real game. There is no point in testing a MockGame. You will not need a mock game, this is like your `main` class.

File: spec/mocks/game_start.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require_relative 'game'
+require_relative 'prompter'
+require_relative 'view'
+
+puts "Hi"
+
+# view = View.new
```
Comment (tommcgee) [inline spec/mocks/game_start.rb]: Should this be commented out? You only want Pull Requests to contain a working app with nothing but production code.

File: spec/game_spec.rb [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+
   
   
-  
+  describe 'game_handler' do
+    it 'will receive printer from mock_view' do
+      expect(mock_view).to receive :printer
+      game.game_handler
+    end
+    it 'will receive game_greeting from prompter' do
```
Comment (tommcgee) [inline spec/game_spec.rb]: missing a new line :)

**Review by tommcgee — COMMENTED** [review]
Other than those minor things, looks good! :+1:

File: spec/prompter_spec.rb [hunk 4]
```diff
@@ -1,14 +1,23 @@
 require_relative '../prompter'
 
 describe 'Prompter' do
-
+
```
Comment (NicoleCarpenter) [inline spec/prompter_spec.rb]: 👾

**Review by NicoleCarpenter — APPROVED** [review]

**Pull request merged**